src/app/enums/virtualization.enum.ts

```diff
@@ -1,6 +1,20 @@
 import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
 import { iconMarker } from 'app/modules/ix-icon/icon-marker.util';
 
+export enum ImageOs {
+  Linux = 'LINUX',
```

Comment:
Values don't have to be uppercase, since this is not a real enum, just a preset for the user.

Reply:
@undsoft - I checked the docs, and there I found these enums:
 
<img width="299" alt="Screenshot 2025-04-30 at 12 36 42" src="https://github.com/user-attachments/assets/93dedc36-624e-447f-94e8-4f59d56f98f1" />